pkg/controller/worker/machine_controller_manager.go

```diff
@@ -16,48 +16,14 @@ package worker
 
 import (
 	"context"
-	"fmt"
-	"path/filepath"
 
-	extensionscontroller "github.com/gardener/gardener/extensions/pkg/controller"
 	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
-	"github.com/gardener/gardener/pkg/utils/chart"
 	kutil "github.com/gardener/gardener/pkg/utils/kubernetes"
-	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
-	rbacv1 "k8s.io/api/rbac/v1"
 
-	"github.com/gardener/gardener-extension-provider-gcp/charts"
 	"github.com/gardener/gardener-extension-provider-gcp/pkg/gcp"
 )
 
-var (
```

Comment:
I think this file can be safely deleted. I did so in https://github.com/gardener/gardener-extension-provider-azure/pull/761 and not many things broke :)

Reply:
Done.